source/server/proto_descriptors.h

```diff
@@ -0,0 +1,4 @@
+#pragma once
+namespace Envoy {
```

Comment:
Sorry I didn't catch this before, please also put this inside the `Server` namespace. (Envoy::Server::validate...)

Reply:
no probs. added now